### PR TITLE
REACT-1.3 ES6 Features.md file added

### LINKED ISSUE #298: FRONTEND WITH FRAMEWORK: REACT-1.3 ES6 Features
Contributors are required to include information about ES6 features, their syntax , an example, a comparison and anything relevant to the framework. 

Also make sure -
Avoid copy/paste, We will be checking.
Changes should be made inside the Frontend_Web_Development_React_Angular_Vue/ dir inside Frontend_Web_Development_React_Angular_Vue branch.
Task will be assigned on first come first serve
Check out [Contribution Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md)
and [Code Of Conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md)

### PR DESCRIPTION
**Pull request by Ravinayak07**
<hr>

## Description 📜
Created the documentation for the issue : REACT-1.3 ES6 Features.
The Documentation includes new features of ES6 , its syntax , examples ,and comparison of React with ES6 and React without ES6 
Fixes #298 

<hr>

## Type of change 📝

- [x] Documentation (Content Creation in the form of codes or tutorials)

<hr>

## Domain of Contribution 📊

- [x] Frontend Dev React/Angular/Vue

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

File: Frontend_Web_Development_React_Angular_Vue/React/Week 1/REACT-1.3 ES6 Features.md [hunk 1]
```diff
@@ -0,0 +1,322 @@
...
+ return ` ${name} ${year}`;
+}
+
+console.log(opensource()); // GWOC 2021
+console.log(opensource('GSSoC')); // GSSoC 2021
+
+```
+<p>&nbsp;</p>
+
+### 6. ***Extracting data from an array***
```
Comment (the-pro) [inline Frontend_Web_Development_React_Angular_Vue/React/Week 1/REACT-1.3 ES6 Features.md:102]: Name it destructuring

File: Frontend_Web_Development_React_Angular_Vue/React/Week 1/REACT-1.3 ES6 Features.md [hunk 2]
```diff
@@ -0,0 +1,322 @@
...
+
+console.log(a, b, c);
+//output
+2 4 6
+
+
+```
+<p>&nbsp;</p>
+
+### 7. ***Extracting data from a object***
```
Comment (the-pro) [inline Frontend_Web_Development_React_Angular_Vue/React/Week 1/REACT-1.3 ES6 Features.md:130]: This can come under destructuring

**Comment by the-pro** [comment]
You can also write a bit about rest operator as you have mentioned spread operator

**Comment by the-pro** [comment]
Rest all looks really nice

**Review by anandhu720 — CHANGES_REQUESTED** [review]
please add changes recommended by @the-pro

**Comment by Ravinayak07** [comment]
> please add changes recommended by @the-pro

Yes sir , working on that. I will submit the changes by today

File: Frontend_Web_Development_React_Angular_Vue/React/Week 1/REACT-1.3 ES6 Features.md (hunk 2 above)
Comment (Ravinayak07) [reply Frontend_Web_Development_React_Angular_Vue/React/Week 1/REACT-1.3 ES6 Features.md]: > You can also write a bit about rest operator as you have mentioned spread operator

sir i have pushed the required changes mentioned by you. Kindly review it.

**Comment by Ravinayak07** [comment]
> please add changes recommended by @the-pro

Done Sir

**Review by anandhu720 — APPROVED** [review]

**Pull request merged**